tests/test_CmFilter.py

```diff
@@ -12,10 +12,10 @@ class test_CmFilter(TestCase):
     def setUp(self):
         test_data = TestDataProvider()
         self.uut = CmFilter(config=test_data.provide_config())
-        with open("./test_data/listing_product_id_27_entries_20.json", "r") as json_test_listing:
+        with open("./test_data/chandra_listing.json", "r") as json_test_listing:
             self.listing = json.load(json_test_listing)
 
-        with open("./test_data/example_card_product_id_27.json", "r") as json_test_card:
+        with open("./test_data/chandra_card.json", "r") as json_test_card:
             self.test_card = json.load(json_test_card)
 
         with open("test_data/listing_product_id_27_filtered_entries_20.json", "r") as json_expected_result:
```

Comment:
I'm not allowed to comment below but shouldn't it be like this:

```suggestion +6
        with open("test_data/listing_product_id_27_filtered_entries_20.json", "r") as json_expected_result:
	            self.expected_result = json.load(json_expected_result)
	            
    def test_prefilter(self):
        prefilter_result = self.uut.prefilter(self.listing, self.test_card)
        self.maxDiff =None
        self.assertDictEqual(prefilter_result, self.expected_result)
```

Reply:
Fixed in commit f2d89156276a25cf693076575c7523a6e1f3a5cf